### PR TITLE
update the pandoc tutorial to reflect the more recent idiom

### PR DESCRIPTION
**Pull request by denten**
Please ask the Spanish translators to reflect these changes. Particularly important is the addition of xelatex, necessary for Spanish.

### CODE REVIEW

**Comment by vgayolrs** [comment]
@denten , Thanks for notifying about the changes. These days I make the adaptation in the Spanish translation.

**Comment by denten** [comment]
Great, let's wait until the editors merge these changes then let me know if I can help with the Spanish diffs.

**Comment by denten** [comment]
Folks, hoping to bump this for a merge while the tutorial is getting some attention on social media. The changes are pretty minimal.

**Comment by acrymble** [comment]
@denten thanks for the updates. As the editor who originally worked on this lesson is no longer part of the team, no one has the level of full intellectual control over its contents to approve a quick change.

Our Spanish team are also working hard on a number of initiatives, and for many people this is also a holiday weekend. We'll certainly have someone take a close look at this, but it may take some time.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [inline en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:467]: While the new command configuration is valid, the [canonical Pandoc documentation](https://pandoc.org/MANUAL.html#using-pandoc) still uses the formulation `pandoc [options] [input-file]...` I hesitate to change this tutorial in order to pull it farther out of sync with Pandoc's actual documentation, and thus cause confusion for users trying to square what we say versus what's in the docs.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [inline en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:509]: Should this read "text editor" instead of "tex editor"?

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [inline en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:510]: Explain to the reader why it's necessary to change the `fontfamily` attribute.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [inline en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:522]: This is a useful feature of Pandoc to note. However, please convert to a footnote since it is a bit of an in-the-weeds sidebar to the main flow of the tutorial - great to have available, but not in a way that disrupts the flow of the lesson.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [inline en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:524]: "In this way..." and the following sentence "One could say..." are a convoluted expression of this aspect of Pandoc's argument parsing. Please tighten up this language.

Also, do you know if the Pandoc docs have a section that lists what subset of the cli args can be specified via YAML and which can't? I did a cursory browse but it wasn't totally clear to me - if you happen to know of a good listing, that'd be nice to add (but then, we might just be stuck with what we've got!)

**Review by mdlincoln — CHANGES_REQUESTED** [review]
Thanks for your patience waiting for this review. I've added my revision comments.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [inline en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:426]: You'll see there's a merge conflict - unfortunately we had to remove the link to Zed Shaw's book because it was giving a 404 response (and also never seems to have been archived in the Wayback Machine, which is normally what we'd try to point to before actually removing a link.)

If you know of where this book moved to, by all means put in a link. We also encourage cross-linking between PH lessons, so an easy (albeit very beginner lesson) is at https://programminghistorian.org/en/lessons/intro-to-bash

**Review by mdlincoln — CHANGES_REQUESTED** [review]

**Comment by vgayolrs** [comment]
Hi @denten . Waiting for the changes to proceed with the editing of the translated lesson.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (denten) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:467]: Good point, though in teaching this tutorial I've now began to advocate for `pandoc [options] input-file -o [output-file]`. The problem with -o is that it must be followed by the output file, making it so that the output path changes position arbitrarily depending on the order of operations. Putting it always as the last argument after the input-file is easier to read and remember. The audience for this tutorial is unlikely to be confused by a more logical order of operations. The majority encounter Pandoc for the first time here, and the few that are familiar with the canonical documentation understand that Shell commands allow for composable syntax. 

In short, I think we should go with `pandoc [options] input-file -o [output-file]` wherever possible because it speaks to "sustainability." I will be happy to revert it if we receive feedback otherwise.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (denten) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:509]: Yes, good catch!

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (denten) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:510]: I wish I knew. It seems to be an arbitrary decision and a source of much confusion in support forums. Nothing that I saw in the documentation explains the reason for this mechanic.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (denten) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:522]: Will do.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (denten) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:426]: Yes, most definitely lets go with the PH tutorial!

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (denten) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:524]: Will clean up the wording. The YAML vs CLI is not always clear. I'll send the reader to `man pandoc` in a footnote.

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:510]: ah sorry, I didn't necessarily mean that we needed a deep explanation 😂  - just add a word or two so that it's clear "when using XeLaTeX for conversion into .pdf, **you must** use `mainfont` instead of `fontfamily`"

File: en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md
Comment (mdlincoln) [reply en/lessons/sustainable-authorship-in-plain-text-using-pandoc-and-markdown.md:467]: I'm convinced! 👍

**Comment by mdlincoln** [comment]
@denten thanks for your feedback on the edits - can you go ahead and push the changes we agreed on? Once that's done, then I think we're good to start the ES translations @vgayolrs

**Comment by mdlincoln** [comment]
Hello @denten - just a reminder that we'll be happy to merge these changes once you've attended to the change requests.

**Comment by denten** [comment]
Yes, thank you, I haven't forgotten. Just trying to hit a deadline with another article. Will try to get to it this weekend.

**Comment by denten** [comment]
Implemented all of the suggested changes in https://github.com/programminghistorian/jekyll/pull/1345

**Pull request closed without merging**
